projects/m2/022-anagrams/js/index.js

```diff
@@ -0,0 +1,26 @@
+// const prompt = require("prompt-sync")({ sigint: true}) ;
+
+function isAnagram(originalText, potentialAnagram){
+    if(originalText.length === potentialAnagram.length){
+        for (let i = 0; i < potentialAnagram.length; i++) {
+            if( ! originalText.includes( potentialAnagram.charAt(i) )){

```

Comment:
Try testing this function using two words: `hello` and `hhhhh`
I expect the result to be **false**, but I believe you will get _true_.
**Fix** the problem ;-)

Reply:
Hi, now the problem is fixed.